RDBM/Table.cpp

```diff
@@ -128,33 +125,53 @@ Table& Table::AddRecord(std::initializer_list<std::string> list)
 	table.push_back(rec);
 	return *this;
 }
-void Table::DeleteRecord(unsigned int  index)
+
+Table& Table::AddColumn(std::string type, std::string name)
+{
+	colname.push_back(name);
+	if (type == "int" || type == "integer")
+		coltype.push_back('i');
+	else if (type == "double")
+		coltype.push_back('d');
+	else
+		coltype.push_back('s');
```

Comment:
I think we need defined name for string column. What if I'll write "dick" in type parameter? It is not string.